backend/qemu.pm

```diff
@@ -231,9 +234,7 @@ sub _wait_for_migrate ($self) {
 
     # Avoid race condition where QEMU allows us to start the VM (set state to
     # running) then tries to transition to post-migarte which fails
-    $self->_wait_while_status_is(qr/paused|finish-migrate/,
-        $max_execution_time - $execution_time,
-        'Timed out waiting for migration to finalize');
+    $self->_wait_while_status_is(qr/paused|finish-migrate/, $max_execution_time - $execution_time, 'Timed out waiting for migration to finalize');
```

Comment:
I wonder if this (and other changes) are result of the extended character count to 160?